charts/lagoon-logging/values.yaml

```diff
@@ -233,9 +233,12 @@ excludeNamespaces:
 # This may require tweaking to handle high volumes of logs.
 clusterOutputBuffer:
   flush_thread_count: 32
-  timekey: 1m
-  timekey_wait: 10s
+  timekey: 10s # create chunks every 10 secs, therefore possible to flush every 10sec
+  timekey_wait: 0s # don't wait additional time for the timekey to be filled
   timekey_use_utc: true
+  flush_thread_burst_interval: '0' # don't sleep if there is more data to flush
```

Comment:
Should this be an integer instead of a string?

Reply:
https://github.com/banzaicloud/logging-operator/blob/master/pkg/sdk/model/output/buffer.go#L72
Apparrently not